recipes/liburing/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+import os
+import shutil
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LiburingConan(ConanFile):
+    name = "liburing"
+    license = "GPL-2.0-or-later"
+    homepage = "https://github.com/axboe/liburing"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    description = """helpers to setup and
+        teardown io_uring instances, and also a simplified interface for
+        applications that don't need (or want) to deal with the full kernel
+        side implementation."""
+    topics = "conan", "asynchronous io"
+
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+
+    _autotools = None
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+
+        self._autotools = AutoToolsBuildEnvironment(self)
+        prefix = self.package_folder
+        conf_args = [
+            "--prefix={}".format(prefix),
+        ]
+        self._autotools.configure(
+            args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(f"{self.name}-{self.name}-{self.version}",
+                  self._source_subfolder)
```

Comment:
```suggestion
        os.rename(
            "{0}-{0}-{1}".format(self.name, self.version),
            self._source_subfolder
        )
```

Be careful, f-strings are not yet supported because we would like to support old Python versions as well.